docs/rum_collection/crash_reporting.md

```diff
@@ -73,32 +78,70 @@ Datadog.initialize(
 Global.rum = RUMMonitor.initialize()
 ```
 
-### Symbolicate reports
+## Symbolicate crash reports
 
-If your iOS error is unsymbolicated, upload your dSYM file using one of the following tools to symbolicate your different stack traces. For any given error, you have access to the file path, the line number, and a code snippet for each frame of the related stack trace.
+Crash reports are collected in a raw format and mostly contain memory addresses. To map these addresses into legible symbol information, Datadog requires dYSM files, which are generated in your application's build or distribution process.
 
-#### Fastlane Plugin
+### Find your dYSM file
+
+Every iOS application produces dYSM files for each application module. Each application version contains a set of dYSM files.
 
-The Datadog plugin helps you upload dSYM files to Datadog from your fastlane configuration.
+Depending on your setup, you may need to download dSYM files from App Store Connect or find them on your local machine. 
+
+| Bitcode Enabled | Description                                                                                                                                                                                                                                                                                       |
+|-----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Yes             | dSYM files are available once [App Store Connect][6] completes processing your application's build.                                                                                                                                                                                                    |
+| No              | Xcode exports dSYM files to `$DWARF_DSYM_FOLDER_PATH` at the end of your application's build. Ensure that the `DEBUG_INFORMATION_FORMAT` build setting is set to **DWARF with dYSM File**. By default, Xcode projects only set `DEBUG_INFORMATION_FORMAT` to **DWARF with dSYM File** in release. |
```

Comment:
```suggestion
| No              | Xcode exports dSYM files to `$DWARF_DSYM_FOLDER_PATH` at the end of your application's build. Ensure that the `DEBUG_INFORMATION_FORMAT` build setting is set to **DWARF with dYSM File**. By default, Xcode projects only set `DEBUG_INFORMATION_FORMAT` to **DWARF with dSYM File** for _Release_ configuration. |
```
Where "Release" is the name of project configuration (like below), so I believe it should be upper cased.

<img width="521" alt="Screenshot 2022-03-14 at 17 49 02" src="https://user-images.githubusercontent.com/2358722/158220829-3a8b83a3-15b7-49bb-9769-27c8d916b3a8.png">